.github/workflows/release.yaml

```diff
@@ -0,0 +1,124 @@
+name: Tests
+
+on:
+  push:
+    tags:
+      - '*'
+  pull_request:  # FIXME: temporary until release.yaml works
```

Comment:
```suggestion
```

Reply:
And thank you so much. This is my first time to setup PyPI release by workflow. It must had taken a lot more time without your help.